### ISSUE TITLE
Remove support for alternate theme engines

### ISSUE THREAD

**Issue by jenlampton** [backdrop/backdrop-issues#438]
In Drupal 7, one theme engine was used. But our theme system is built with the ability to have swapable theme engines. This adds unnecessary complexity and overhead with performance implications.  Let's remove this.

**Comment by BWPanda** [comment]
Adding 'needs more feedback' label so others can chime in (either on doing this at all, or on the best way to do this). If however this was already decided on (and forgotten about?), feel free to remove.

**Comment by klonos** [comment]
I agree. Perhaps this can be left to https://github.com/silkscreencms/silkscreen. @jlfranklin ??
